path/__init__.py

```diff
@@ -317,13 +267,26 @@ def getcwd(cls):
         """
         return cls(getcwdu())
 
+    cwd = classmethod(getcwd)
+
+    @classmethod
+    def home(cls):
+        """ This class method works like pathlib "Path.home" class method.
+
+        :return: a Path object pointing to user home directory.
+        :rtype: Path
+        """
+        return cls(os.path.expanduser('~'))
```

Comment:
Try not to reference `os.path`.

```suggestion
        return cls('~').expanduser()
```